<launch>

  <arg name="UAV_NAME" default="$(env UAV_NAME)"/>

  <group>
    <node pkg="mrs_uav_state_estimation" type="plot_juggler.sh" name="replace_uav_name" output="screen">
    </node>

    <node pkg="plotjuggler" type="plotjuggler" name="plotjuggler" args="-l /tmp/manager_debug.xml" launch-prefix="bash -c 'sleep 1; $0 $@' ">
    </node>
  </group>

</launch>
